app/scripts/modules/core/domain/serverGroup.ts

```diff
@@ -12,28 +12,30 @@ interface IAsg {
 }
 
 export interface ServerGroup {
-  account: string;
+  account?: string;
   app?: string;
   asg?: IAsg;
   buildInfo?: any;
   category?: string;
-  cloudProvider: string;
-  cluster: string;
+  cloudProvider?: string;
+  cluster?: string;
   entityTags?: IEntityTags;
   detail?: string;
   executions?: Execution[];
-  instanceCounts: InstanceCounts;
+  instanceCounts?: InstanceCounts;
   instanceType?: string;
-  instances: Instance[];
+  instances?: Instance[];
+  isDisabled?: boolean;
   launchConfig?: any;
   loadBalancers?: string[];
-  name: string;
+  name?: string;
   provider?: string;
-  region: string;
+  region?: string;
   runningTasks?: ITask[];
   searchField?: string;
+  securityGroups?: string[];
   stringVal?: string;
   tags?: any;
-  type: string;
+  type?: string;
   vpcName?: string;
 }
```

Comment:
I think the only valid changes here are adding `securityGroups?: string[]` and `isDisabled?: boolean`. The rest should probably be reverted.